src/shogun/multiclass/KDTreeKNNSolver.h

```diff
@@ -26,7 +26,11 @@ class CKDTREEKNNSolver : public CKNNSolver
 {
 	public:
 		/** default constructor */
-		CKDTREEKNNSolver() { init(); }
+		CKDTREEKNNSolver() 
+		{
+			CKNNSolver::init(); 
```

Comment:
same, why dont you just call the default ctor in of `CKNNSolver` and call in that the `init()`, why explicitly like this?